benchmarking/harness.py

```diff
@@ -145,6 +147,11 @@ def _getInfo(self):
             info["commands"][getArgs().framework] = {
                 "backend": getArgs().backend
             }
+        if getArgs().wipe_cache:
+            if "commands" not in info:
+                info["commands"] = {getArgs().framework: {}}
+            info["commands"][getArgs().framework]["wipe_cache"] = \
```

Comment:
Here we may encounter the case that the commands exist in info but framework does not exist in info["commands"]

Reply:
I assume the info["command"] is first created in line 146 or 152. You meant there is an existing info["command"]?